spec/shared_behaviors/taggable.rb

```diff
@@ -29,6 +34,11 @@ def tag_link(s)
       expect(formatted_string).to include(tag_link('what'))
       expect(formatted_string).to include(tag_link('hey'))
       expect(formatted_string).to include(tag_link('vöglein'))
+      expect(formatted_string).to include(tag_link('ինչո՞ւ'))
+      expect(formatted_string).to include(tag_link('այո՜ո'))
+      expect(formatted_string).to include(tag_link('սեւ֊սպիտակ'))
+      expect(formatted_string).to include(tag_link('գժանո՛ց'))
+      expect(formatted_string).to include(tag_link('മലയാണ്മ'))
     end
```

Comment:
Also make sure the extraction works correctly by adding cases to the `it 'extracts tags despite surrounding text'` spec.